### PR TITLE
Main menu

### PR DESCRIPTION
**Pull request by miryamfv**
Responsive main menu with NFT totals from proxy API

### CODE REVIEW

File: components/main-menu.module.scss [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    color: $color-accent;
+    text-transform: uppercase;
+    font-size: 16px;
+    margin-left: 0;
+}
+
+.secondary {
+
+    border: 1px solid $color-white;
+    background-color: $color-mine-shaft;
```
Comment (lorenzopolidori) [inline components/main-menu.module.scss:87]: This outlined button has been updated to have transparent background in the design.

**Review by lorenzopolidori — CHANGES_REQUESTED** [review]

File: components/layout.module.scss [hunk 2]
```diff
@@ -10,19 +10,75 @@
...
+        @media (max-width: $screen-small) {
+            padding: 0 20px;
+        }
+    }
+}
+
+.notconnected {
+    z-index: $z-index-navbar;
+     padding: 15px 0;
+     border-bottom: 1px solid $color-alto;
```
[the remark this answers is not in the document]
Comment (miryamfv) [reply components/layout.module.scss:27]: I have it enabled

File: components/layout.js [hunk 3]
```diff
@@ -19,13 +32,22 @@ export default function Layout({ children }) {
                     content={''} />
                 <meta name={'twitter:card'} content={'summary_large_image'} />
             </Head>
-            <header className={styles.navbar}>
+            <header className={!user.authenticated ? `${styles.notconnected}` : `${styles.navbar}`}>
                 <div className={styles.navContent}>
+                    <div className={styles.logowrapper}>
                     <Logo className={styles.logo} />
-                    <TopMenu />
+                    <span>TESTNET</span>
```
[the remark this answers is not in the document]
Comment (miryamfv) [reply components/layout.js:39]: I do have it but I will pull now

File: components/layout.module.scss [hunk 4]
```diff
@@ -10,19 +10,75 @@
     opacity: 0.9;
     padding: 15px 0;
     background-color: $color-mine-shaft;
+    border-bottom: 1px solid $color-alto;
```
[the remark this answers is not in the document]
Comment (miryamfv) [reply components/layout.module.scss:13]: should I change the yellow or leave it? I grabbed the exact yellow from the logo svg

File: components/main-menu.js [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+export default function MainMenu({ visible }) {
+    const [accountNfts, setAccountNfts] = useState({})
+    const { onMint, onToggleMenu } = useContext(MenuEventContext)
+    const user = useContext(UserContext)
+    
+    useEffect(() => {
+        const fetchNfts = async () => {
+            
+            try{
```
[the remark this answers is not in the document]
Comment (miryamfv) [reply components/main-menu.js:18]: It is enabled since before starting changes yesterday. but  I just `git pulled origin dev`

**Review by miryamfv — COMMENTED** [review]
I have done the changes I pulled the dev branch and now I get an error from `./components/drop-down-selector.tsx:1:0`:
`Cannot find module 'uniqid' or its corresponding type declarations.ts(2307)`
Looking on readme maybe Im missing a configuration?

File: components/top-menu.js [hunk 6]
```diff
@@ -4,24 +4,33 @@ import { useContext } from 'react'
...
+                        
+                        {/* <li><</li> */}
+                        <li><button className={styles.brand} onClick={onToggleMenu}>{maskWalletAddress(user.walletAddress)} | {user.walletBalance} ALGO</button></li>
+                        <li className={styles.toggle_menu} onClick={onToggleMenu} ><button><i className={mainMenuVisible ? 'icon-cross' : 'icon-menu'} /></button></li>
                     </>
                 }
                 {!user.authenticated &&
-                    <li className={styles.brand} onClick={onConnectWallet}>{strings.connectWallet}</li>
+                    <>
+                    <li><button className={styles.brand} onClick={onConnectWallet}>{strings.connectWallet}</button></li>
```
[the remark this answers is not in the document]
Comment (miryamfv) [reply components/top-menu.js:26]: fixed

File: components/layout.module.scss (hunk 4 above)
[the remark this answers is not in the document]
Comment (lorenzopolidori) [reply components/layout.module.scss:13]: `border-bottom` should be `1px solid #545454`, although `#545454` should be defined in `colors.scss` with a memorable name.

File: components/layout.module.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (lorenzopolidori) [reply components/layout.module.scss:28]: That's odd, if you save the file pressing `CMD+S` or `CTRL+S`, does VSCode not format the file?

**Review by lorenzopolidori — APPROVED** [review]

**Pull request merged**